common/src/test/java/io/druid/common/guava/GuavaUtilsTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.common.guava;
+
+import junit.framework.Assert;
+import org.junit.Test;
+
+import java.util.Objects;
+
+public class GuavaUtilsTest
+{
+  @Test
+  public void testParsLong()
+  {
+    Assert.assertTrue(Objects.equals(Long.parseLong("+100"), GuavaUtils.tryParseLong("+100")));
```

Comment:
Should be `Assert.equals` or `Assert.<Object>equals` so that the values actually get printed out in the failure mesasges